packages/big-design/src/components/Input/Input.tsx

```diff
@@ -141,9 +141,13 @@ const StyleableInput: React.FC<InputProps & PrivateProps> = ({
     };
 
     if (Array.isArray(error)) {
-      error.forEach(validateError);
+      const nextError = error.reduce<Array<Props['error']>>((acc, errorItem) => {
```

Comment:
This type is saying that it could be an `Array<React.ReactNode> | Array<Array<React.ReactNode>>`. I think we could just change it to this:
```suggestion
      const nextError = error.reduce<Array<React.ReactNode>>((acc, errorItem) => {
```